opm/input/eclipse/EclipseState/InitConfig/Equil.hpp

```diff
@@ -6,8 +6,15 @@
 
 namespace Opm {
     class DeckKeyword;
+    class DeckRecord;
+
     class EquilRecord {
         public:
+            EquilRecord() = default;
+            EquilRecord( double datum_depth_arg, double datum_depth_pc_arg, double woc_depth, double woc_pc, double goc_depth, double goc_pc, bool live_oil_init, bool wet_gas_init, int target_accuracy, bool humid_gas_init);
+            EquilRecord(const DeckRecord& record);
```

Comment:
Could you mark the `DeckRecord` constructor as `explicit` please?

Reply:
done.